examples/architectureComponentsExample/src/main/java/io/realm/examples/arch/BackgroundTask.java

```diff
@@ -16,66 +16,73 @@
 
 package io.realm.examples.arch;
 
-import android.arch.lifecycle.ViewModel;
-import android.os.SystemClock;
+import android.annotation.SuppressLint;
 import android.support.annotation.MainThread;
+import android.support.annotation.NonNull;
+import android.util.Log;
 
 import io.realm.Realm;
 import io.realm.RealmResults;
 import io.realm.examples.arch.model.Person;
 
 
-public class Updater extends ViewModel {
+public class BackgroundTask {
+    private static final String TAG = "BackgroundTask";
+
+    private boolean isStarted;
 
     private volatile Thread thread;
 
+    @MainThread
+    public boolean isStarted() {
+        return isStarted;
+    }
+
     @MainThread
     public void start() {
```

Comment:
All of these methods depend on `start` only being called from a single thread. It would be more clear and safer if start/stop was synchronized.